docs/guide.md

```diff
@@ -2593,6 +2641,18 @@ There are two ways to create an immutable **List**, **Set**, **Bag**, **Stack**
 <!-- -->
 
 ```java
+
+// 1. Calling toImmutable()
+ImmutableList<String> list = 
+    Lists.mutable.with("a", "b", "c").toImmutable(); // creates a list of elements a,b,c
+ImmutableSet<String> set0 =  
+    Lists.mutable.with("a", "b", "a", "c").toSet().toImmutable(); // creates a set of elements a,b,c
+ImmutableSet<String> set1 =  
+    Sets.mutable.with("a", "b", "a", "c").toImmutable(); // creates a set of elements a,b,c
+ImmutableMap<Integer, String> map = 
+    Maps.mutable.with(1, "a", 2, "b", 3, "c").toImmutable(); // creates a map with keys 1,2,3
+
+//2. Using factory classes
 ImmutableList<String> emptyList_i = 
     Lists.immutable.empty();  // creates an empty list
 ImmutableList<String> list_b = 
```

Comment:
While you are updating the documentation, please add methods to create `ImmutableSet` and `ImmutableMap` using factory classes. New users might get confused if that is not referred. Thank you!